diff --git a/dist/path.js b/dist/path.js
index 9afa4f1f74c15eeaff0044b289332939c2287b11..71002d4b8072c2a414c7fd24d7ee64070cae810d 100644
--- a/dist/path.js
+++ b/dist/path.js
@@ -1,8 +1,9 @@
 // oxlint-disable-next-line no-restricted-imports
 import nodePath from 'node:path';
-import ts from 'typescript';
 export function slash(path) {
-    return ts.server.toNormalizedPath(path);
+	// We use TypeScript 7 which does not provide tsserver.
+	// An simple version of the `toNormalizedPath` function is inlined in our patch to allow css module kit to function.
+    return path.replace(/\\/g, "/");
 }
 export function join(...paths) {
     return slash(nodePath.join(...paths));
diff --git a/src/path.ts b/src/path.ts
index 174a6ac865a7a6b88dab6c02ef9a8b671a9367b8..79810c10dd2f7e1b88f0f0646d8716edd7d1ee43 100644
--- a/src/path.ts
+++ b/src/path.ts
@@ -2,10 +2,11 @@
 import type { ParsedPath } from 'node:path';
 // oxlint-disable-next-line no-restricted-imports
 import nodePath from 'node:path';
-import ts from 'typescript';
 
 export function slash(path: string): string {
-  return ts.server.toNormalizedPath(path);
+	// We use TypeScript 7 which does not provide tsserver.
+	// An simple version of the `toNormalizedPath` function is inlined in our patch to allow css module kit to function.
+    return path.replace(/\\/g, "/");
 }
 
 export function join(...paths: string[]): string {
